src/Aspire.Hosting.Python/PublicAPI.Unshipped.txt

```diff
@@ -1,6 +1,6 @@
 #nullable enable
-Aspire.Hosting.Python.PythonProjectResource
```

Comment:
@joperezr - do you know why all these APIs are still under "Unshipped" when we released an 8.2 version of these APIs?

The Shipped file says:

https://github.com/dotnet/aspire/blob/62f8cf792deee900b8afd04c8e9e278c7dc6fa8a/src/Aspire.Hosting.Python/PublicAPI.Shipped.txt#L1

Reply:
Oh yeah I was going to ask that lol